desktop/index.md

```diff
@@ -30,50 +34,85 @@ Docker Desktop includes:
-
-The **Volumes** view displays a list of volumes and allows you to easily create and delete volumes and see which ones are being used. For more information, see [Explore volumes](use-desktop/volumes.md).
-
-In addition, the Docker Dashboard allows you to:
-
-- Easily navigate to the **Preferences** (**Settings** in Windows) menu to configure Docker Desktop preferences. Select the **Preferences** or **Settings** icon in the Dashboard header.
-- Access the **Troubleshoot** menu to debug and perform restart operations. Select the **Troubleshoot** icon in the Dashboard header.
-
-### Other key features of Docker Desktop:
+</div>
+<div id="features" class="tab-pane fade" markdown="1">
 
 * Ability to containerize and share any application on any cloud platform, in multiple languages and frameworks.
 * Easy installation and setup of a complete Docker development environment.
 * Includes the latest version of Kubernetes.
-* Automatic updates to keep you up to date and secure.
 * On Windows, the ability to toggle between Linux and Windows Server environments to build applications.
 * Fast and reliable performance with native Windows Hyper-V virtualization.
 * Ability to work natively on Linux through WSL 2 on Windows machines.
 * Volume mounting for code and data, including file change notifications and easy access to running containers on the localhost network.
-* In-container development and debugging with supported IDEs.
-
-## Download and install
-
-Docker Desktop is available for Mac, Linux, and Windows. For download information, system requirements, and installation instructions, see:
-
 
-* [Install Docker Desktop on Linux](install/linux-install.md)
-* [Install Docker Desktop on Mac](install/mac-install.md)
-* [Install Docker Desktop on Windows](install/windows-install.md)
+</div>
+</div>
 
-For information about Docker Desktop licensing, see [Docker Desktop License Agreement](../subscription/index.md#docker-desktop-license-agreement).
+Docker Desktop works with your choice of development tools and languages and
+gives you access to a vast library of certified images and templates in
+[Docker Hub](https://hub.docker.com/). This enables development teams to extend
+their environment to rapidly auto-build, continuously integrate, and collaborate
+using a secure repository.
 
-For information about new features, improvements, and bug fixes in Docker Desktop releases, see [Release notes](release-notes.md).
+<div class="component-container">
+    <!--start row-->
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+             <div class="component-icon">
+                 <img src="/images/download-docker.svg" alt="Docker Desktop for Mac" width="70" height="70">
+             </div>
```

Comment:
I wonder if we can include these elements in our stylesheets; from the homepage / landing page, it looks like we already have some icons that we use for "download", "reference", etc., so possibly it would be good to re-use the same icons/elements for consistency;

<img width="500" alt="Screenshot 2022-08-28 at 19 39 09" src="https://user-images.githubusercontent.com/1804568/187087442-42d7b425-213a-4611-a095-cf837eeb9c6c.png">